onnxruntime/test/testdata/custom_op_library/custom_op_library.cc

```diff
@@ -80,12 +80,12 @@ struct KernelOne {
   }
 
  private:
-  OrtApi api_;  // keep a copy of the struct, whose ref is used in the ort_
+  const OrtApi& api_;  // keep a copy of the struct, whose ref is used in the ort_
```

Comment:
nit: fix or remove comment
also in KernelTwo

Reply:
Even better, I removed the api_ member entirely. It was never used.